src/layout/mod.rs

```diff
@@ -1583,6 +1586,14 @@ impl<W: LayoutElement> Layout<W> {
         ws_idx == mon.active_workspace_idx
     }
 
+    pub fn urgent_window(&mut self, id: &W::Id) {
```

Comment:
I don't think this is needed here, along with `LayoutElement::set_urgent()`. Since the layout code doesn't need to set urgency, it's always done externally where it can just use a `Mapped` method.